src/Sylius/Bundle/LocaleBundle/Templating/Helper/LocaleHelper.php

```diff
@@ -27,19 +27,30 @@
     private $localeConverter;
 
     /**
+     * @var string
+     */
+    private $defaultLocale;
+
+    /**
      * @param LocaleConverterInterface $localeConverter
+     * @param string $defaultLocale
      */
-    public function __construct(LocaleConverterInterface $localeConverter)
+    public function __construct(LocaleConverterInterface $localeConverter, string $defaultLocale)
     {
         $this->localeConverter = $localeConverter;
+        $this->defaultLocale = $defaultLocale;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function convertCodeToName(string $localeCode): ?string
+    public function convertCodeToName(string $localeCode, ?string $locale = null): ?string
```

Comment:
`locale` -> `resultLocale`/`displayLocale`. It is quite ambiques to have `localeCode` and `locale`

Reply:
I agree and would go with displayLocale !